app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -37,58 +41,33 @@ EventAdapter createEventAdapter() {
     }
 
     @Override
-    void fetchEvents() {
-        final DateTime now = DateTime.now();
-        DateTime start = now.minusMonths(2).dayOfMonth().withMinimumValue();
-        DateTime end = now.plusYears(1).dayOfMonth().withMaximumValue();
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        mPlusId = getArguments().getString(Const.EXTRA_PLUS_ID);
+        mCacheKey = "event_" + mPlusId;
+    }
 
+    @Override
+    void fetchEvents() {
         setIsLoading(true);
-        final String plusId = getArguments().getString(Const.EXTRA_PLUS_ID);
-        final String cacheKey = "event_" + plusId;
 
         if (Utils.isOnline(getActivity())) {
-            App.getInstance().getGdgXHub().getChapterEventList(plusId, start, end).enqueue(
-                new Callback<PagedList<Event>>() {
-                    @Override
-                    public void success(PagedList<Event> eventsPagedList) {
-                        List<Event> events = eventsPagedList.getItems();
-                        splitEventsAndAddToAdapter(events);
-                        App.getInstance().getModelCache().putAsync(cacheKey,
-                            mEvents,
-                            DateTime.now().plusHours(2),
-                            new ModelCache.CachePutListener() {
-                                @Override
-                                public void onPutIntoCache() {
-                                    mAdapter.addAll(mEvents);
-                                    setIsLoading(false);
-                                }
-                            });
-                    }
-
-                    @Override
-                    public void failure(Throwable error) {
-                        onError(R.string.fetch_events_failed);
-                    }
-
-                    @Override
-                    public void networkFailure(Throwable error) {
-                        onError(R.string.offline_alert);
-                    }
-                });
+            onListLoadMore(1, 0);
```

Comment:
`onListLoadMore(1, 0)` alone is not that understandable. Can you extract this into a separate method sayin something like `loadFirstPageOfEvents()`